learning-team/2020-06/2020-06-04.md

```diff
@@ -0,0 +1,118 @@
+# June 4, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [x] Chris Manson (CM)
+- [x] Amy Lam (AL)
+- [x] Ricard Mendes (RM)
+- [ ] Jessica Jordan (JJ)
+- [x] Melanie Sumner (MS)
+- [x] Jen Weber (JW)
+- [ ] Kenneth Larsen (KL)
+- [x] Robert Wagner (RW)
+- [x] Jared Galanis (JG)
+- [x] Isaac Lee (IL)
+
+### Guests
+
+- [x] Chris Krycho (CK)
+
```

Comment:
I noticed that we had an Alex today. Not sure who they were, however.

```suggestion
- [x] Alex

```

Reply:
not sure

Reply:
Was it https://github.com/void-mAlex? From https://github.com/emberjs/core-notes/pull/309

Reply:
Sorry I forgot to update! It was, they confirmed on Discord

Reply:
got it thanks, ready for review if one of you can approve and merge!